### ISSUE TITLE
restore pretrained ckpt failed

### ISSUE THREAD

**Issue by yang502** [DetectionTeamUCAS/R2CNN-Plus-Plus_Tensorflow#51]
@yangxue0827 I download your pretrained ckpt file (R2CNN_20180922_DOTA_v28/voc_280002model.ckpt), but meet the following errors? Do you have some tips to solve this problem?
===========Error hints like this:

`NotFoundError (see above for traceback): Restoring from checkpoint failed. This is most likely due to a Variable name or other graph key that is missing from the checkpoint. Please ensure that you have not altered the graph expected based on the checkpoint. Original error:

Key resnet_v1_50/block1/unit_1/bottleneck_v1/conv1/BatchNorm/beta not found in checkpoint
	 [[node save/RestoreV2 (defined at ../libs/networks/build_whole_network.py:618)  = RestoreV2[dtypes=[DT_FLOAT, DT_FLOAT, DT_FLOAT, DT_FLOAT, DT_FLOAT, ..., DT_FLOAT, DT_FLOAT, DT_FLOAT, DT_FLOAT, DT_FLOAT], _device="/job:localhost/replica:0/task:0/device:CPU:0"](_arg_save/Const_0_0, save/RestoreV2/tensor_names, save/RestoreV2/shape_and_slices)]]
	 [[{{node save/RestoreV2/_397}} = _Recv[client_terminated=false, recv_device="/job:localhost/replica:0/task:0/device:GPU:0", send_device="/job:localhost/replica:0/task:0/device:CPU:0", send_device_incarnation=1, tensor_name="edge_397_save/RestoreV2", tensor_type=DT_FLOAT, _device="/job:localhost/replica:0/task:0/device:GPU:0"]()]]
`

**Comment by yang502** [comment]
Sorry, it seems that I use ResNet50 model to load your pretrained model, I change the network to ResNet101, the problems solved.
